exporter/cortexexporter/README.md

```diff
@@ -0,0 +1,15 @@
+Exporter to a Cortex instance, or any Prometheus Remote Write compatible backend.
```

Comment:
Readme should have information for an end-user of the exporter as well right? Not just info on the code?

Reply:
Totally agree. We will add those before pushing to upstream. We are thinking of adding introduction and a sample configuration. This is a internal draft.